parser/src/test/scala/jawn/JNumIndexCheck.scala

```diff
@@ -15,12 +15,12 @@ class JNumIndexCheck extends Properties("JNumIndexCheck") {
       def add(s: CharSequence): Unit = ()
       def add(v: Boolean): Unit =
         if (!v) failed = true
-      def finish: Boolean = !failed
+      def finish(): Boolean = !failed
     }
 
-    val singleContext: RawFContext[Boolean] = new JNumIndexCheckContext(false)
-    val arrayContext: RawFContext[Boolean] = new JNumIndexCheckContext(false)
-    val objectContext: RawFContext[Boolean] = new JNumIndexCheckContext(true)
+    def singleContext(): RawFContext[Boolean] = new JNumIndexCheckContext(false)
+    def arrayContext(): RawFContext[Boolean] = new JNumIndexCheckContext(false)
+    def objectContext(): RawFContext[Boolean] = new JNumIndexCheckContext(true)
```

Comment:
Why do we have to change these vals into defs?

Reply:
Dotty's pickier about mixing `()` and no `()`. We could change the definitions in `Facade` but I was trying to minimize changes (see [this thread](https://github.com/typelevel/jawn/pull/212#discussion_r363835235) for some other discussion).